src/flashhls-adapter.js

```diff
@@ -92,6 +106,17 @@ class FlashHLSAdapter extends FakeEventTarget {
     if (this._el && this._el.parentNode) {
       this._el.innerHTML = '';
     }
+    this._startTimeAttach = NaN;
+    this._lastTimeDetach = NaN;
+    delete this._api;
```

Comment:
```suggestion
this._api = null;
```
Just change the type to `?FlashAPI`